docs/docs/deployment.md

```diff
@@ -2,35 +2,26 @@
 
 ## Foreword
 
-As of beginning 2016 the docker toolset is not quite there to provide a heroku like experience
-when deploying to production. A lot of parts are already there, but have a few quirks that need
-to be addressed.
-
-Because of that, the deployment strategy for djangopackages.org is a bit different from what you
-read in the getting started with docker tutorials.
-
-First, we don't use docker-machine. It's not reliable and has no team of maintainers comparable
-to other distros like debian, ubuntu or rhel that manages security releases.
-
-Second, there's no way do daemonize the docker compose process. When the underlying VM is
-restarted, the stack won't start automatically.
+djangopackages.org use Docker Compose both for local development and in production.
 
 The current strategy is:
 
-> - Use a virtual machine with a well patched OS (debian, ubuntu, RHEL), djangopackages.org is using
->   ubuntu 14.04
-> - Install docker, docker-compose, git and supervisord
-> - Clone the code on the server
-> - Let supervisord run it
+- Use a virtual machine with a well patched OS (debian or ubuntu), djangopackages.org is using
```

Comment:
This just kind of trails off and doesn't indicate what djangopackages.org is using.

Reply:
We are more tied to an app platform than a host OS these days so I'm not sure we have a set host requirement. The only reason I haven't tried running it on DO's app platform is that it would be more expensive. Open to ideas though.